kind/hack/populate_minio.sh

```diff
@@ -1,20 +1,45 @@
 #!/usr/bin/env bash
 
-echo "creating minio alias"
-mc alias set kfp http://localhost:8080 minio minio123
-echo "creating test bucket"
-mc mb kfp/test
-echo "copying data"
-# code modules
-mc cp --recursive ${ROOT_DIR}/../transforms/code/code_quality/test-data/input/ kfp/test/code_quality/input
-mc cp --recursive ${ROOT_DIR}/../transforms/code/proglang_select/test-data/input/ kfp/test/proglang_select/input
-mc cp --recursive ${ROOT_DIR}/../transforms/code/proglang_select/test-data/languages/ kfp/test/proglang_select/languages
-mc cp --recursive ${ROOT_DIR}/../transforms/code/malware/test-data/input/ kfp/test/malware/input
-# universal
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/doc_id/test-data/input/ kfp/test/doc_id/input
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/ededup/test-data/input/ kfp/test/ededup/input
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/fdedup/test-data/input/ kfp/test/fdedup/input
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/filter/test-data/input/ kfp/test/filter/input
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/noop/test-data/input/ kfp/test/noop/input
-mc cp --recursive ${ROOT_DIR}/../transforms/universal/tokenization/test-data/ds01/input/ kfp/test/tokenization/ds01/input
+# Define variables
+minio_url="http://localhost:8080"
+minio_alias="kfp"
+minio_access_key="minio"
+minio_secret_key="minio123"
+source_dir="${ROOT_DIR}/../transforms"
 
+# Function to copy data
+copy_data() {
+    local module="$1"
+    local src="$2"
+    local dest="$3"
+
+    echo "Copying data for $module"
+    mc cp --recursive "${source_dir}/${module}/test-data/input/" "kfp/test/${dest}/input"
+}
+
+# Set Minio alias
+echo "Creating Minio alias"
+mc alias set "$minio_alias" "$minio_url" "$minio_access_key" "$minio_secret_key"
+
+# Create test buckets
+echo "Creating test buckets"
```

Comment:
we don't need a separate bucket for each tested transformer. The "folder" prefix is enough.